config/features.rb

```diff
@@ -9,10 +9,6 @@
             description: "A feature only used by tests; not to be used for any actual features."
   end
 
-  feature :design_system_downtime_new,
```

Comment:
There's still a reference to this feature in `test/integration/downtime_integration_test.rb`.